### PR TITLE
deathknight: blood pve fixes.

### PR DESCRIPTION
**Pull request by arcadepro**

### CODE REVIEW

File: data/Deathknight.lua [hunk 1]
```diff
@@ -107,8 +108,8 @@ lib:__RegisterSpells('DEATHKNIGHT', 80100, 1, {
 					 48792, -- Icebound Fortitude
 					 55233, -- Vampiric Blood (Blood)
 					 81256, -- Dancing Rune Weapon (Blood)
-					194679, -- Rune Tap (Blood talent)
-					287081, -- Lichborne (Frost/Unholy honor talent)
+					194679, -- Rune Tap (Blood)
+					 49039, -- Lichborne
```
Comment (Rainrider) [inline data/Deathknight.lua:112]: Nit: this one should be the third one in the survival list (sort by spell id asc)

File: data/Deathknight.lua [hunk 2]
```diff
@@ -183,7 +184,7 @@ lib:__RegisterSpells('DEATHKNIGHT', 80100, 1, {
 	[ 91837] =  63560, -- Putrid Bulwark (Ghoul) (Unholy) <- Dark Transformation
 	[ 91838] =  47484, -- Huddle (Ghoul) (Unholy)
 	[101568] = 178819, -- Dark Succor (Frost/Unholy)
-	[145622] =  51052, -- Anti-Magic Zone (honor talent)
+	[145629] =  51052, -- Anti-Magic Zone
```
Comment (Rainrider) [inline data/Deathknight.lua:187]: Nit: sorting is off here as well

File: data/Deathknight.lua [hunk 3]
```diff
@@ -20,15 +20,16 @@ along with LibPlayerSpells-1.0. If not, see <http://www.gnu.org/licenses/>.
 
 local lib = LibStub('LibPlayerSpells-1.0')
 if not lib then return end
-lib:__RegisterSpells('DEATHKNIGHT', 80100, 1, {
+lib:__RegisterSpells('DEATHKNIGHT', 80100, 2, {
 	COOLDOWN = {
-		  46584, -- Raise Dead (Unholy)
+		  46585, -- Raise Dead
+		  46584, -- Raise Dead (Rank 2 - Unholy)
```
Comment (Rainrider) [inline data/Deathknight.lua:26]: Nit: sorting off as well here

**Comment by Rainrider** [comment]
Sorry for the nits, just trying to stick to conventions. Not sure how much this actually brings but it sure looks tidy this way :) It is something Adirelle wanted btw, before someone blames it on my CDO :)

**Comment by arcadepro** [comment]
No problem.

**Comment by Rainrider** [comment]
Thanks again mate!

**Pull request merged**